core/src/main/scala/org/apache/spark/sql/delta/commands/VacuumCommand.scala

```diff
@@ -498,9 +498,45 @@ trait VacuumCommandImpl extends DeltaCommand {
       basePath: Path,
       relativizeIgnoreError: Boolean,
       isBloomFiltered: Boolean): Seq[String] = {
-    getActionRelativePath(action, fs, basePath, relativizeIgnoreError).map { relativePath =>
+    val paths = getActionRelativePath(action, fs, basePath, relativizeIgnoreError).map {
+      relativePath =>
```

Comment:
```suggestion
    val paths = getActionRelativePath(action, fs, basePath, relativizeIgnoreError)
      .map { relativePath =>
```
Bit clearer what goes with what like this imo.